spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper'
+
+describe UsersController do
+  
+  describe "GET #new" do
+    it "assigns a new User to @user" do
+      get :new
+      expect(assigns(:user)).to be_a_new(User)
+    end
+
+    it "renders the :new template" do
+      get :new
+      expect(response).to render_template(:new)
+    end
+  end
+
+  describe "POST #create" do
+    context "with valid attributes" do
+      before { @new_valid_user = Fabricate.attributes_for(:user) }
```

Comment:
I believe you can use `let` here like so `let(:new_valid_user) { Fabricate.attributes_for(:user) }`